ldms/src/sampler/lustre_client/lustre_client_general.c

```diff
@@ -98,7 +116,8 @@ int llite_general_schema_init(const char *producer_name)
 err2:
         ldms_schema_delete(sch);
 err1:
-        log_fn(LDMSD_LERROR, SAMP" lustre_llite_general schema creation failed\n");
+        log_fn(LDMSD_LERROR, SAMP" lustre_llite_general schema creation failed with: %s\n",
```

Comment:
If every conditional has its own error message, then I don't think we should print a second one at the end.

Reply:
The second one at the end centralizes the decoding of STRERROR.

Reply:
Yes, but lets just do that in the new log messages. I am strongly opposed to splitting an error message across multiple log entries unless absolutely necessary. In this case it is not at all necessary.

Reply:
fixed.